change/@azure-msal-browser-2f59d691-955b-4fc1-831c-ce6b66d252e7.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Refactor AccountEntity into type",
```

I think adding these PR numbers still make changes easier to find if any bugs come up.

```suggestion
  "comment": "Refactor AccountEntity into type #7674",
```